changelogs/unreleased/gh-6920-fix-crash-in-on-replace-triggers.md

```diff
@@ -0,0 +1,4 @@
+## bugfix/core
+
+* Ban ddl operations from space on_replace triggers, since
```

Comment:
Nit:
 - [Release notes should be Past Simple](https://www.tarantool.io/en/doc/latest/contributing/release_notes/).
 - DDL is an abbreviation so it should be in CAPs.
 - 'from' doesn't sound right here. Should be 'in'.

```
* Banned DDL operations in space on_replace triggers, since they could lead
  to a crash (gh-6920).
```

Reply:
Fixed